app/Decsys/Services/FriendlyIdService.cs

```diff
@@ -0,0 +1,39 @@
+using Decsys.Utilities;
+
+namespace Decsys.Services;
+
+public class FriendlyIdService
```

Comment:
Actually, instead of this being a Service that needs injecting, can we make it (and all its methods `static`?

If it's possible to make all the methods static (it should be) I think it'll work better as a static Utility Class (so move it to `Utilities`and rename it to something like `FriendlyIds` or `FriendlyIdHelper` or `FriendlyIdUtility`.

Then you can get rid of the dependency injection bits for it too.